Resources/Prototypes/Guidebook/medical.yml

```diff
@@ -28,7 +28,6 @@
   name: guide-entry-chemist
   text: "/ServerInfo/Guidebook/Medical/Chemist.xml"
   children:
-  - Medicine
```

Comment:
?

Reply:
The `Medicine` guide is already a child of the  `Chemicals` guide, which appears earlier on, so it currently just logs an error about a duplicate entry anytime the guide gets opened and drops it.

Having the same guide appear multiple times in the guidebook ToC / tree in different places isn't supported atm, but I guess I should just comment it out for now and leave a TODO. Or we could change it so that it is supported, but if feels kind of weird?

Reply:
TODO is probably fine for now. I wanted the ability for guides to appear multiple times in the tree, but some of the others didn't. So it's up the air whether to implement that or not.

Reply:
I don't feel strongly either way, but to me it feels more natural to treat the tree/children like a book's ToC